relay-dynamic-config/src/global.rs

```diff
@@ -183,14 +183,6 @@ pub struct Options {
     )]
     pub http_span_allowed_hosts: Vec<String>,
 
-    /// Whether or not relay should drop attachments submitted with transactions.
-    #[serde(
-        rename = "relay.drop-transaction-attachments",
-        deserialize_with = "default_on_error",
-        skip_serializing_if = "is_default"
-    )]
-    pub drop_transaction_attachments: bool,
```

Comment:
Instead of dropping the attachments, we now splot the transaction and its attachments and send them to different topics. This flag was never enabled.